addons/certificate/utils.py

```diff
@@ -0,0 +1,10 @@
+import base64
+
+
+def _get_formatted_value(data, formatting='encodebytes'):
+    if formatting == 'encodebytes':
+        return base64.encodebytes(data)
+    elif formatting == 'base64':
+        return base64.b64encode(data)
+    else:
+        return data
```

Comment:
Imo, it's a bit overkill to create an utility file just for a single function you don't use outside of your new module